resippy/utils/proj_utils.py

```diff
@@ -0,0 +1,36 @@
+import osr
+from pyproj import Proj
+import utm
+
+
+def wkt_to_proj_string(wkt,            # type: str
+                ):              # type: (...) -> Proj
+    spatialref = osr.SpatialReference()
+    spatialref.ImportFromWkt(wkt)
+    proj_string = spatialref.ExportToProj4()
+    return proj_string
+
+
+def wkt_to_proj(wkt,            # type: str
+                ):              # type: (...) -> Proj
+    proj_string = wkt_to_proj_string(wkt)
+    proj = Proj(proj_string)
+    return proj
+
+
+def wkt_to_osr_spatialref(wkt,  # type: str
+                          ):              # type: (...) -> Proj
+    spatialref = osr.SpatialReference()
+    spatialref.ImportFromWkt(wkt)
+    return spatialref
+
+
+def decimal_degrees_to_local_utm_proj(lon_decimal_degrees,      # type: float
+                                      lat_decimal_degrees,      # type: float
+                                      ):                        # type: (...) -> Proj
+    zone_info = utm.from_latlon(lat_decimal_degrees, lon_decimal_degrees)
+    zone_string = str(zone_info[2]) + zone_info[3]
+    zone_letter = zone_string[-1]
+    is_south_zone = zone_letter < 'N'
```

Comment:
Should this logic be switched since `'S' < 'N'` returns `False`

Reply:
Not that simple, and I'm sure it's confusing.  Zone letters aren't a simple "N" or "S", they're based on this:
http://www.dmap.co.uk/utmworld.htm
Anything south of the "N" line is in the south zone, another north of the "N" line is in the north zone.  And yes, the Zone letter "S" is north of "S", pretty stupid.